src/redux/orders/orders.operations.js

```diff
@@ -0,0 +1,185 @@
+            cost {
+              currency
+              value
+            }
+          }
+          address {
+            country
+            region
+            city
+            zipcode
+            street
+            buildingNumber
+            appartment
+          }
+          items {
+            category {
+              lang
+              value
+            }
+            subcategory {
+              lang
+              value
+            }
+            model {
+              lang
+              value
+            }
+            name {
+              lang
+              value
+            }
+            colors {
+              lang
+              value
+            }
+            pattern {
+              lang
+              value
+            }
+            closure {
+              lang
+              value
+            }
+            closureColor
+            size {
+              heightInCm
+              widthInCm
+              depthInCm
+              volumeInLiters
+              weightInKg
+            }
+            bottomMaterial {
+              lang
+              value
+            }
+            bottomColor {
+              lang
+              value
+            }
+            additions {
+              lang
+              value
+            }
+            actualPrice {
+              currency
+              value
+            }
+            quantity
+          }
+          totalItemsPrice {
+            currency
+            value
+          }
+          totalPriceToPay {
+            currency
+            value
+          },
+          paymentMethod,
+          isPaid
+        }
+      }
+    `
+  });
+  const { data } = result;
+  return data.getAllOrders;
+};
+
+const getOrderById = (id) => {
+  const query = `
+    query getOrder($id: ID){
+      getOrderById(id: $id) {
+        ...on Order {
+          _id
+          status
+          user {
+            firstName
+            lastName
+            patronymicName
+            email
+            phoneNumber
+          }
+          dateOfCreation
+          lastUpdatedDate
+          adminComment
+          userComment
+          cancellationReason
+          delivery {
+            sentOn
+            sentBy
+            invoiceNumber
+            courierOffice
+            byCourier
+            cost {
+              currency
+              value
+            }
+          }
+          address {
+            country
+            region
+            city
+            zipcode
+            street
+            buildingNumber
+            appartment
+          }
+          items {
+            name {
+              lang
+              value
+            }
+          }
+          totalItemsPrice {
+            currency
+            value
+          }
+          totalPriceToPay {
+            currency
+            value
+          }
+          isPaid
+          paymentMethod
+        }
+        ...on Error {
+          statusCode
+          message
+        }
```

Comment:
When you provide a fragment in the query you also need to provide handling for it in case you receive an `Error` object as a response for reference check `business-pages` or `news` operation methods

Reply:
fixed